.github/workflows/pylint.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint Python Code with Pylint
+
+# Runs on push and pull requests to the master branch only
+on:
+  push:
+   branches: "master"
```

Comment:
The push events should be every time a commit occurs across all branches.

use the following below to do that

`**`